group_vars/postgresql/production.yml

```diff
@@ -33,14 +33,14 @@ datadog_typed_checks:
     configuration:
       init_config:
       instances:
-        - dbm: true
-          host: localhost
+        - host: localhost
+          dbm: true
           port: 5432
           username: datadog
           password: "{{ vault_datadog_db_password }}"
       logs:
         - type: file
-          path: /var/lib/postgresql/{{ postgres_version }}/main/pg_log/*.csv
+          path: /var/lib/postgresql/{{ postgres_version }}/main/pg_log/pg.json
```

Comment:
I ran this branch against a staging server to confirm the correct path and name for the log after the config changes in this PR go in. Events are logged to `pg.json`. A file called `pg.log` does exist, but it simply says: 
```
2023-10-04 21:19:49.790 UTC [3606944]     651dd6f5.3709a0 LOG:  ending log output to stderr
2023-10-04 21:19:49.790 UTC [3606944]     651dd6f5.3709a0 HINT:  Future log output will go to log destination "jsonlog"
```